how-to.md

```diff
@@ -242,6 +242,13 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
         - Under Configuration Settings, click Computer Configuration on the left tab then select Mozilla > Firefox > Cookies Select “Do not allow preferences to be changed" > on the right panel, select “Enabled" > OK
         - Finish the creation steps
 - Run a Sync to export the newly added ADMX resource and profile to Simeon
+- On Azure DevOps (https://dev.azure.com/) > Tenants > Repos > navigate to Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyUploadedDefinitionFiles
```

Comment:
@david-simeon, probably not now, but we should consider getting rid of all DevOps specific instructions. What about clients using GitHub?